packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -246,7 +246,7 @@ export function PanelChrome({
         <>
           <HoverWidget
             menu={menu}
-            title={title}
+            title={typeof title === 'string' ? title: undefined}
```

Comment:
Both here and on line 249, `title` is used in `PanelMenu` to generate a more descriptive `aria-label`. Using a `ReactElement` instead of a string means we can't use it in `aria-label`, which means it will be generic `Menu for panel with no title` 🤔